src/components/lineup/types.ts

```diff
@@ -0,0 +1,108 @@
+import { ID, UID } from 'audius-client/src/common/models/Identifiers'
+import Kind from 'audius-client/src/common/models/Kind'
+import { Lineup as LineupData } from 'audius-client/src/common/models/Lineup'
+import { LineupActions } from 'audius-client/src/common/store/lineup/actions'
+import { Maybe } from 'audius-client/src/common/utils/typeUtils'
+import { SectionListProps } from 'react-native'
+
+export enum LineupVariant {
+  MAIN = 'main',
+  SECTION = 'section',
+  CONDENSED = 'condensed',
```

Comment:
I'm not sure we have all these types on mobile. I think just "main" and "playlist" are the interesting ones